src/components/select-field/__tests__/__snapshots__/BaseSelectField-test.js.snap

```diff
@@ -5,7 +5,7 @@ exports[`components/select-field/BaseSelectField renderSelectButton() should sen
   aria-activedescendant={null}
   aria-autocomplete="list"
   aria-expanded={false}
-  aria-owns="selectfield11"
+  aria-owns="selectfield12"
```

Comment:
For this change I'm not sure if it should be here. It looks like aria-owns is referring to the selectFieldID which is generated via uniqueId generator.